assets/scss/7-layout/_col-grid.scss

```diff
@@ -0,0 +1,33 @@
+// Column grid (l-col-grid)
+//
+// Just testing the waters of a tiny grid helper. For now, used on photo grids and in AMP templates. By default the grid is an even two column grid. We include variations for col counts and spacing.
+//
+// l-col-grid--1x3 - Columns split in thirds
+// l-col-grid--xxxs-gap - Shrink up the default gap. Used on photo grid
+// l-col-grid--resp - Collapse all columns at `bp-m`
+//
+// Markup: 7-layout/col-grid.html
+//
+// Styleguide 7.0.1
+//
+.l-col-grid {
+  @supports (display: grid) {
+    @include gap;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+
+    &--1x3 {
+      grid-template-columns: repeat(3, 1fr);
+    }
+
+    &--xxxs-gap {
+      @include gap($size-xxxs);
+    }
+    
+    @include mq($until: bp-m) {
+      &--resp {
```

Comment:
I think it's likely we'll add other responsive variations to this class. Given that, it might be good to be a little more specific with this modifier -- maybe even keep it consistent with some of our other breakpoint-based naming, e.g. `&--stack-from-bp-l` or something.

Reply:
It's funny, I did that at first but then I thought nah too long. I can change that back tho, no biggie.